rsapi/auth.go

```diff
@@ -184,13 +185,20 @@ type oAuthSigner struct {
 	refreshAt    time.Time
 	client       httpclient.HTTPClient
 	accountID    int // optional, only used to set X-Account header if present
+	flexeraOne   bool
 }
 
 // Sign adds the OAuth bearer header to the *http.Request
 func (s *oAuthSigner) Sign(req *http.Request) error {
 	if time.Now().After(s.refreshAt) {
 		jsonStr := fmt.Sprintf(`{"grant_type":"refresh_token","refresh_token":"%s"}`, s.refreshToken)
-		authReq, err := http.NewRequest("POST", buildURL(s.host, "api/oauth2"),
+		host := s.host
+		url := "api/oauth2"
+		if s.flexeraOne {
+			host = "login.flexera.com"
```

Comment:
This will not work correctly if someone is trying to use `rsc` against Flexera One EU.

Reply:
Discussed with @douglaswth on teams we are going to support EU in a future release of better in Flexera One CLI.